### DISCUSSION TITLE
How many items should a player be able to carry?

### DISCUSSION THREAD

**Discussion by ShadowCommander** [space-wizards/space-station-14#21255] [Ideaguys]
What items do you expect to be able to carry as each role?

**Comment by ShadowCommander** [comment]
Full set of tools for deconstructing windows with only a backpack as a passenger.

**Comment by Just-a-Unity-Dev** [comment]
Engineers should be expected to carry cables, steel and rods in their bags. Additionally plasteel and glass but those first 3 are the main thing I see most engineers carry.
